site/src/api/typesGenerated.ts

```diff
@@ -281,6 +363,18 @@ export interface Healthcheck {
   readonly threshold: number
 }
 
+// From codersdk/flags.go
+export interface IntFlag {
+  readonly name: string
+  readonly flag: string
+  readonly env_var: string
+  readonly shorthand: string
+  readonly description: string
+  readonly enterprise: boolean
+  readonly default: number
+  readonly value: number
```

Comment:
I don't really understand why we'd have flag types that aren't booleans. What would be the use case for an `IntFlag`, for instance? I see it used in `DeploymentFlags` but I'm not getting it.

Reply:
So for example UserWorkspaceQuota, if we had a bool instead we would then need to do a fetch call to see what the value was from some other endpoint. From an int flag we can tell it's both enabled (greater that 0) and has some value we care about. This assumption is probably rooted in the golang behavior that an empty int is 0, an empty string is "", etc. Same with an empty string, we know when it's set and what the value is without a second fetch for the data.

Reply:
Cool, thanks for explaining!